### DISCUSSION TITLE
how to automatically go to terminal mode when focus on terminal pane?

### DISCUSSION THREAD

**Discussion by cStralpt** [vscode-neovim/vscode-neovim#1838] [Q&A]
what i want to achieve is to send some keybindings only when in terminal mode in my vscode
```
  {
    "key": "ctrl+/",
    "command": "vscode-neovim.send",
    "when": "terminalFocus && neovim.mode == terminal",
    "args": "<C-/>"
  },
```

and then in my neovim config it will do this:
```
	map("t", "<C-/>", function()
    print("invoked!")
		callVSCodeFunction("call VSCodeCall('workbench.action.terminal.focus')")
	end, { noremap = true, silent = true, desc = "toggle terminal" })

```

NOTE: i will make a function to make the terminal toggleable by invoking this:
`workbench.action.closePanel`

but the problem is in VSCode Neovim it don't go into terminal mode when open a tterminal/when focus in tterminal.

**Comment by xiyaowong** [comment]
Not possible.
